### DISCUSSION TITLE
Convenience function to get URL from endpoint

### DISCUSSION THREAD

**Discussion by agusterodin** [reduxjs/redux-toolkit#1813] [Ideas]
It would be nice to be able to generate a URL from an endpoint for situations where you want the URL without using RTK query

Something like

apiEndpoints.getAllSubscriptionDetails.getUrl() which would return a string with the URL

**Comment by phryneas** [comment]
RTK Query is transport mechanism agnostic - how would that look if the transport mechanism is graphql, grpc or SQL?

**Reply by agusterodin** [reply]
> RTK Query is transport mechanism agnostic - how would that look if the transport mechanism is graphql, grpc or SQL?

True, I guess if someone really wanted to make the URL reusable they could just create a constant or function in their endpoints file.

Ended up not needing to do this anyway.

Thanks for the help good sir.
